corehq/apps/reports/cache.py

```diff
@@ -12,7 +13,7 @@ class _ReportQuickCache(QuickCache):
     """
     def __call__(self, *args, **kwargs):
         report = args[0]
-        if report.is_cacheable and _is_valid(report):
+        if report.is_cacheable and _is_valid(report) and getattr(settings, 'CACHE_REPORTS', True):
```

Comment:
nit but if we're adding this to the main `settings.py` any reason not to use the standard `settings.CACHE_REPORTS`?